frontend/src/v5/ui/routes/dashboard/teamspaces/teamspaceLayout/teamspaceLayout.component.tsx

```diff
@@ -33,7 +34,8 @@ interface ITeamspaceLayout {
 
 export const TeamspaceLayout = ({ children, className }: ITeamspaceLayout): JSX.Element => {
 	const { teamspace } = useParams<TeamspaceParams>();
-	const user = CurrentUserHooksSelectors.selectCurrentUser();
+	const DEFAULT_TEAMSPACE_IMG_SRC = 'assets/images/teamspace_placeholder.svg';
+	const teamspaceImgSrc = generateV5ApiUrl(`teamspaces/${teamspace.name}/avatar?${Date.now()}`, clientConfigService.GET_API);
```

Comment:
Would move this to a file as a helper function. imageUrls.helper.ts maybe? also shove  'getThumbnailBasicPath' from 'src\v5\ui\controls\formSelectView\formSelectView.component.tsx' in there.

Reply:
Done